Swiftfin/Components/LetterPickerBar/LetterPickerBar.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+struct LetterPickerBar: View {
+
+    @ObservedObject
+    var viewModel: FilterViewModel
+    private var onSelect: (FilterCoordinator.Parameters) -> Void
```

Comment:
Not used, can remove.

Reply:
Just the onSelect or both the onSelect AND the viewModel? I have the onSelect removed but let me know if that viewModel shouldn't be there. I have it as part of my init so I would need a bit of a rework.

Reply:
Just `onSelect` and everything that refers to it.